### ISSUE TITLE
Replace emoji with description

### ISSUE THREAD

**Issue by bmabir17** [bsolomon1124/demoji#9]
**Is your feature request related to a problem? Please describe.**
I am trying to replace the emoji found with description given in the `demoji.findall(tweet)`
{
    "🔥": "fire",
    "🌋": "volcano",
    "👨🏽\u200d⚖️": "man judge: medium skin tone",
    "🎅🏾": "Santa Claus: medium-dark skin tone",
    "🇲🇽": "flag: Mexico",
    "👹": "ogre",
    "🤡": "clown face",
    "🇳🇮": "flag: Nicaragua",
    "🚣🏼": "person rowing boat: medium-light skin tone",
    "🐂": "ox",
}
**Describe the solution you'd like**
I have a string with emoji : 
Rooney ! Oh dear, oh dear ! Fucking dreadful 🙈⚽️⚽️

I want to make it like this:
Rooney ! Oh dear, oh dear ! Fucking dreadful  'see-no-evil monkey' 'soccer ball'  'soccer ball'

**Describe alternatives you have considered**
```python
emo_dict=demoji.findall(emoticonString)
for key in emo_dict.keys():
    emoticonString=emoticonString.replace(key, emo_dict[key])
```

**Comment by davidagud** [comment]
@bmabir17 I just created a pull request with a function that would allow for this, but it would look like "Rooney ! Oh dear, oh dear ! Fucking dreadful :see-no-evil monkey: :soccer ball: :soccer ball:

**Comment by bmabir17** [comment]
@davidagud Thank you for the pull request https://github.com/bsolomon1124/demoji/pull/10

**Comment by davidagud** [comment]
@bmabir17 @bsolomon1124 Just created a pull request with the updated code.

**Comment by bsolomon1124** [comment]
Closed in PR #12 .
